lib/grandfathered-descriptions.js

```diff
@@ -130,6 +130,7 @@ module.exports = [
   'data-store',
   'datazenit',
   'dbglass',
+  'debugtron',
```

Comment:
Please add comment why he was added. This case is an exception than default action.

Reply:
OK. Comment added